pomegranate/hmm.pyx

```diff
@@ -1279,6 +1279,119 @@ cdef class HiddenMarkovModel(GraphModel):
             return [emissions, sequence_path]
         return emissions
 
+    def sample_state(self, state_probs, length=1):
+        """Generate samples from a model in a given emission state
+
+        Returns a list of feature vectors.
+        The model must have been baked first in order to run this method.
+
+        Parameters
+        ----------
+        state_probs : array-like, shape (n_states)
+            The normalized probabilities of being in each state such as those
+            returned from predict_proba
```

Comment:
@jmschrei  Note that predict_proba returns a subset of states (the visible ones).  Do you think a `sample_state` should do that as well?